python/docs/jsonschema_tutorial.rst

```diff
@@ -0,0 +1,180 @@
+    :dedent:
+
+And pack the points using the Python built-in ``struct`` and ``base64`` libraries.
+
+.. literalinclude:: ../examples/jsonschema/pointcloud_csv_to_mcap.py
+    :start-after: # tutorial-pack-points-start
+    :end-before: # tutorial-pack-points-end
+    :dedent:
+
+We set an identity pose to place our point cloud at the center of the scene, and add an arbitrary
+``frame_id``.
+
+.. literalinclude:: ../examples/jsonschema/pointcloud_csv_to_mcap.py
+    :start-after: # tutorial-pose-frame-id-start
+    :end-before: # tutorial-pose-frame-id-end
+    :dedent:
+
+We'll leave the ``timestamp`` field for later, when we write the messages into the MCAP file.
+
+Writing the MCAP file
+---------------------
+
+Now that the point cloud is built, we can write it into an MCAP file. We'll start with some imports
+from the `Python MCAP library <https://github.com/foxglove/mcap/tree/main/python>`_:
+
+.. literalinclude:: ../examples/jsonschema/pointcloud_csv_to_mcap.py
+    :start-after: # tutorial-mcap-imports-start
+    :end-before: # tutorial-mcap-imports-end
+    :dedent:
+
+Let's open a file where we'll write our output MCAP data. First, we'll need to write our header.
+
+.. literalinclude:: ../examples/jsonschema/pointcloud_csv_to_mcap.py
+    :start-after: # tutorial-write-header-start
+    :end-before: # tutorial-write-header-end
+    :dedent:
+
+.. note::
+  We can choose whatever name we want for ``library``. We're not using one of the
+  `MCAP well-known profiles <https://github.com/foxglove/mcap/blob/main/docs/specification/appendix.md#well-known-profiles>`_,
+  so we use our own custom profile name. Custom profile names must begin with ``x-``.
+
+Next, create a "channel" of messages to contain our point cloud. The schema
+name and content informs Foxglove Studio that it can parse and display this message as a point
+cloud.
+
+.. literalinclude:: ../examples/jsonschema/pointcloud_csv_to_mcap.py
+    :start-after: # tutorial-write-channel-start
+    :end-before: # tutorial-write-channel-end
+    :dedent:
+
+Next, we write our messages. If we only wrote one message, our MCAP file would have an infinitely
+short duration. To address that, we write our point cloud message a few times with increasing
+timestamps.
+
+.. literalinclude:: ../examples/jsonschema/pointcloud_csv_to_mcap.py
+    :start-after: # tutorial-write-message-start
+    :end-before: # tutorial-write-message-end
+    :dedent:
+
+Finally we can invoke ``finish()`` on the MCAP writer. This will include the summary and footer
+in the output file.
+
+.. literalinclude:: ../examples/jsonschema/pointcloud_csv_to_mcap.py
+    :start-after: # tutorial-finish-start
+    :end-before: # tutorial-finish-end
+    :dedent:
+
+That's it! We now have a valid MCAP file with 10 point cloud messages. We can
+check this with the `MCAP CLI tool <https://github.com/foxglove/mcap/tree/main/go/cli/mcap>`_:
+
+.. code-block:: bash
+
+    $ brew install mcap
```

Comment:
Maybe break up the install and the info commands?

i.e. "Install the MCAP CLI tool using the following command... then, do xyz".